profile/src/main/java/com/scottlogic/deg/profile/validators/profile/constraints/atomic/TemporalConstraintValidator.java

```diff
@@ -19,20 +22,32 @@ public TemporalConstraintValidator(String rule, List<FieldDTO> fields)
     public final ValidationResult validate(TemporalConstraintDTO dto)
     {
         ValidationResult fieldMustBeValid = fieldMustBeValid(dto);
-        if(!fieldMustBeValid.isSuccess) return fieldMustBeValid;
+        if (!fieldMustBeValid.isSuccess) return fieldMustBeValid;
 
-        ValidationResult dateMustBeSpecified = dateMustBeSpecified(dto);
-        if(!dateMustBeSpecified.isSuccess) return dateMustBeSpecified;
+        ValidationResult dateTimeMustBeValid = dateTimeMustBeValid(dto);
+        if (!dateTimeMustBeValid.isSuccess) return dateTimeMustBeValid;
 
-        return valueMustBeValid(dto, FieldType.DATETIME);
+        return fieldTypeMustMatchValueType(dto, FieldType.DATETIME);
     }
 
-
-    private ValidationResult dateMustBeSpecified(TemporalConstraintDTO dto)
+    private ValidationResult dateTimeMustBeValid(TemporalConstraintDTO dto)
     {
-        String date = dto.getDate();
-        return date != null && !date.isEmpty()
-            ? ValidationResult.success()
-            : ValidationResult.failure("Date must be specified" + getErrorInfo(dto));
+        String dateTime = dto.getDate();
+        if (dateTime == null || dateTime.isEmpty())
+        {
+            return ValidationResult.failure("DateTime must be specified" + getErrorInfo(dto));
+        }
+        try
+        {
+            OffsetDateTime offsetDateTime = DateTimeFactory.create(dateTime);
+            if (offsetDateTime.getYear() > 9999 || offsetDateTime.getYear() < 1)
```

Comment:
These should be pulled from `com.scottlogic.deg.common.util.Defaults` to avoid duplication.

Reply:
Addressed